cmd/shortener/config/config.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"flag"
+	"strings"
+
+	"github.com/caarlos0/env/v10"
+)
+
+// var errInvalidFormat error = errors.New("address must be in format 'host:port'. i.e. localhost:8080")
+
+const (
+	defaultServerAddress   = ":8080"
+	defaultResponseAddress = "http://localhost:8080"
+)
+
+type Config struct {
+	ServerAddress   string `env:"SERVER_ADDRESS"`
+	ResponseAddress string `env:"BASE_URL"`
+}
+
+func (cfg *Config) Load() {
+	cfg.ServerAddress = defaultServerAddress
+	cfg.ResponseAddress = defaultResponseAddress
+	cfg.parseFlags()
+	cfg.parseEnv()
+}
+
+func (cfg *Config) parseEnv() {
+	env.Parse(cfg)
```

Comment:
What if we get an error?

Reply:
Since we have defaults, the only solution I came up with is to log an error and continue